### ISSUE TITLE
Pretty print feature as development aid

### ISSUE THREAD

**Issue by paterczm** [lightblue-platform/lightblue-core#298]
I'd like to be able to enable JSON formatting (pretty print) for all lightblue responses in the dev environment. That would make working with lightblue more convenient.

**Comment by paterczm** [comment]
Another solution is using a browser plugin (e.g. https://addons.mozilla.org/en-us/firefox/addon/jsonview/).

**Comment by alechenninger** [comment]
Once the data management app is updated, that should help. Issues to watch for that:

- https://github.com/lightblue-platform/lightblue-applications/pull/54
- https://github.com/lightblue-platform/lightblue-rest/issues/75

**Comment by jewzaam** [comment]
Pretty printing can be done by the client.   As you noted, there are browser plugins.  For command line:

```
curl http://lightblue/rest/whatever | python -mjson.tool
```

I don't think there's a real need to do this on the server, but I'll leave this issue open for a while for comment.

**Comment by paterczm** [comment]
I agree this is of low importance, since there are so many good ways to pretty print json from the client side.
